README.md

```diff
@@ -1,44 +1,47 @@
 ## How to Use This Stack
 
 1. Install VirtualBox
- - I've successfully used 4.2.22, 4.3.8, 4.3.12, 4.3.18
+ - I've successfully used 4.2.22, 4.3.8, 4.3.12, 4.3.18, 6.0.8
 2. Install VirtualBox Extension Pack
  - This is necessary to be able to PXE boot a VM without a iso file or
    using the default Intel network adapter
  - I had sucess on VBox 4.2.22 using the PCnet-FAST III (Am79C973)
    adapter without the extension pack but others reported issues to me
-2. Install Vagrant
-  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now.
-3. `vagrant plugin install oscar`
-  - Optional: `vagrant plugin install vagrant-multiprovider-snap`
-4. `vagrant up`
-  - Optional: `vagrant snap take`
-4. You can ssh into any of the boxes with `vagrant ssh <box_name>`
+3. Install Vagrant
+  - I've personally used 1.4.3, 1.6.3 in the past and 1.6.5 right now. Version
+2.2.4 also works.
+4. `vagrant plugin install oscar`
+  - Optional: `vagrant plugin install vagrant-multiprovider-snap`; this is not
```

Comment:
I would just remove the instructions for adding multiprovider-snap.  `vagrant snapshot` has been built in long enough we could just set a pre-req of using vagrant 1.8+

Reply:
Updated.